app/server.js

```diff
@@ -243,6 +243,56 @@ app.get("/quote", (req, res) => {
     });
 })
 
+app.get("/price", (req, res) => {
+    let ticker = req.query.symbol;
+    getQuote(ticker).then((response) => {
+        if (response.status === 200) {
+            var data = response.data;
+            res.json(data);
+        } else {
+            console.log("Message: " + response.data.error);
+        }
+    }).catch((error) => {
+        console.log(error);
+    });
+})
+
+app.post("/buy", (req, res) => {
+    let ticker = req.body.symbol;
+    let action = req.body.action;
```

Comment:
can you make action uppercase?

Reply:
if someone makes a curl post request with action 'buY' the database will be messed up